SyntheticsRunTestsTask/resolve-config.ts

```diff
@@ -134,7 +133,7 @@ export const resolveConfig = async (reporter: synthetics.MainReporter): Promise<
 }
 
 export const getReporter = (): synthetics.MainReporter => {
-  const context: BaseContext = {
+  const context = {
```

Comment:
This seems like an unrelated change, what's the reason here?

Reply:
This type constraint started to have a TS error because of the new version of node installed by `volta`.
Upgrading `@types/node` fixed the TS error, but I thought it was unnecessary to import `clipanion` here.

I'll take of this in my ongoing refactor though of the CI extensions though. So let's revert this change.